README.md

```diff
@@ -309,43 +307,6 @@ skate.define('my-component', {
 
 
 
-#### `events`
-
-Event listeners to add to the custom element. These get bound after the `prototype` is set up and before `created` is called.
-
-```js
-skate.define('my-component', {
-  events: {
-    click (elem, eventObject) {}
-  }
-});
-```
-
-The arguments passed to the handler are:
-
-- `elem` is the DOM element
-- `eventObject` is the native event object that was dispatched on the DOM element
-
-
-
-##### Event Delegation
-
-Event descriptors can use selectors to target descendants using event delegation.
-
-```js
-skate.define('my-component', {
-  events: {
-    'click button' (elem, eventObject) {}
-  }
-});
-```
-
-Instead of firing for every click on the component element - or that bubbles to the component element - it will only fire if a descendant `<button>` was clicked.
-
-Event delegation works with or without a shadow root as it will inspect the event `path` if it exists.
-
-
-
```

Comment:
As a consumer, it would be good to replace this section with a guide for how to use the `on*` syntax (doesn't appear to be documented at length anywhere else in the README, thought its mentioned in the `skate.link` section).